include/nlohmann/detail/json_ref.hpp

```diff
@@ -57,7 +57,7 @@ class json_ref
 
     value_type const* operator->() const
     {
-        return &**this;
+        return &** this;
```

Comment:
result of "make amalgamate"

Reply:
I forgot to check in the amalgamated version. Please update your branch.